File: src/main/java/cn/solarmoon/immersive_delight/common/block_entity/PlateBlockEntity.java
package cn.solarmoon.immersive_delight.common.block_entity;

import cn.solarmoon.immersive_delight.common.block_entity.base.AbstractPlateBlockEntity;
import cn.solarmoon.immersive_delight.common.registry.IMBlockEntities;
import net.minecraft.core.BlockPos;
import net.minecraft.world.level.block.state.BlockState;

public class PlateBlockEntity extends AbstractPlateBlockEntity {
    public PlateBlockEntity(BlockPos pos, BlockState state) {
        super(IMBlockEntities.PLATE.get(), pos, state);
    }
}

File: src/main/kotlin/cn/solarmoon/idyllic_food_diary/element/matter/food_container/plate/PlateBlockEntity.kt
package cn.solarmoon.idyllic_food_diary.element.matter.food_container.plate

import cn.solarmoon.idyllic_food_diary.element.matter.food_container.FoodContainerBlockEntity
import cn.solarmoon.idyllic_food_diary.registry.common.IFDBlockEntities
import net.minecraft.core.BlockPos
import net.minecraft.world.level.block.state.BlockState

class PlateBlockEntity(pos: BlockPos, state: BlockState): FoodContainerBlockEntity(IFDBlockEntities.PLATE.get(), 9, pos, state) {
}